SPDYUnitTests/SPDYURLRequestTest.m

```diff
@@ -314,6 +318,41 @@ - (void)testSPDYProperties
     STAssertEquals(immutableCopy.SPDYBypass, (BOOL)TRUE, nil);
     STAssertEquals(immutableCopy.SPDYBodyStream, stream, nil);
     STAssertEquals(immutableCopy.SPDYBodyFile, @"Bodyfile.json", nil);
+    STAssertEquals(immutableCopy.SPDYURLSession, urlSession, nil);
+}
+
+- (void)testRequestCopyDoesRetainProperties
+{
+    NSURL *url = [[NSURL alloc] initWithString:@"http://example.com/test/path"];
+    NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:url];
+    NSInputStream *stream = [[NSInputStream alloc] initWithData:[NSData new]];
+    NSURLSession *urlSession = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration]];
+
+    request.SPDYBodyStream = stream;
+    request.SPDYBodyFile = @"Bodyfile.json";
+    request.SPDYURLSession = urlSession;
+
+    NSInputStream __weak *weakStream = stream;
+    NSURLSession __weak *weakURLSession = urlSession;
+    NSMutableURLRequest __weak *weakRequest = request;
+    NSURLRequest *immutableCopy = [request copy];
+
+    @autoreleasepool {
```

Comment:
I was specifically referring to the autoreleasepool, it doesn't seem like the complier would general any autorelease calls simply because you set these variables to nil.